homeassistant/components/nam/sensor.py

```diff
@@ -132,6 +136,18 @@
         device_class=SensorDeviceClass.CO2,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    SensorEntityDescription(
+        key=ATTR_SDS011_CAQI,
+        name="SDS011 CAQI",
+        native_unit_of_measurement="CAQI",
```

Comment:
The AQI device class should not have a unit.

https://developers.home-assistant.io/docs/core/entity/sensor

If this measurement is using the CAQI index we shouldn't use the AQI device class as these indices use different scales.

Reply:
Yes you're right. CAQI is very similar to AQI, the calculation is a bit different, but the values are similar and mean the same. I have removed the unit for this sensor.

Reply:
The CAQI index has a range of 100 while AQI has a range of 500. I don't think we should use device class AQI for CAQI as the values can't be compared.

Reply:
I have removed `device_class`.